README.md

```diff
@@ -5,7 +5,7 @@ applications written in server-side JavaScript.
 
 ## Documentation
 
-See the [Circle API docs](https://developers.circle.com/reference/ping?lang=node) for Node.js.
+See the [Circle API docs](https://developers.circle.com/reference) for Node.js.
```

Comment:
We don't have any docs for our SDK. Should we just get rid of this section?

Reply:
Yea, we can get rid of the sentence. Maybe we can still keep the link and move it to the first sentence: 
"The Circle Node library provides convenient access to the Circle API for applications written in server-side JavaScript. For the API reference, see the [Circle API docs](https://developers.circle.com/reference)"